### ISSUE TITLE
Module is not exporting typescript types

### ISSUE THREAD

**Issue by planktoon1** [awslabs/aws-jwt-verify#40]
The module is built with typescript but is not exporting any types which makes it hard to work within a typescript project. 

Specifically, I want to use the verifier inside a class where I initialize it in the constructor of the class and assign it to a private member. 
But I'm missing the `CognitoJwtVerifierSingleUserPool` type to be able to do it.

```
class AuthMiddleware {
private verifier: CognitoJwtVerifierSingleUserPool<{
    userPoolId: string;
    tokenUse: "access";
    clientId: string;
}>;

... 

this.verifier = CognitoJwtVerifier.create({
  userPoolId: "<user_pool_id>",
  tokenUse: "access",
  clientId: "<client_id>",
});
...
```

**Comment by ottokruse** [comment]
HI @planktoon1,
Makes sense, think we should add some `export` statements to make that easier.
In the meantime here is a workaround:

```typescript
import { CognitoJwtVerifier } from "aws-jwt-verify";

class Test {
  verifier: CognitoJwtVerifier<
    {
        // add only properties here, that you're not passing in the actual `create` call, e.g.
        // clientId: string
        // properties that you add here, will become mandatory on individual `verify` calls (i.e. make the 2nd paramater mandatory)
    },
    {
      issuer: string;
      audience: null;
      userPoolId: string;
    }, // just satisfy the type, for assignment below you don't have to spell out everything
    false // false for single user pool, true for multi user pool
  >;
  constructor() {
    this.verifier = CognitoJwtVerifier.create({
      userPoolId: "<user_pool_id>",
      tokenUse: "access",
      clientId: "<client_id>",
    });
  }
  public async test() {
    await this.verifier.verify("ey......");
  }
}
```

**Comment by ottokruse** [comment]
Solved by #42
